site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md

```diff
@@ -0,0 +1,364 @@
+Duration: 5
+
+Under the status for the connector, which displays "Choose Resources", select **Configure**.
+
+This displays the Configure Connector dialog. By default, the fields are set to the names of objects that are created when you configure the connector.
+
+![default config](assets/configuredefaults.png)
+
+Check out [Configuring the Snowflake Connector for ServiceNow](https://other-docs.snowflake.com/en/connectors/servicenow/servicenow-installing-ui.html#configuring-the-snowflake-connector-for-servicenow) for more information on these fields. 
+
+Select **Configure**. The dialog box closes and the status of the connector changes to Provisioning. It can take a few minutes for the configuration process to complete.
+
+> aside negative
+> Watch out!!! The created warehouse is created as a **Large** and with a auto timeout of 10 minutes. So this means, if you set to refresh every hour, the Large warehouse (8 credits/hour) will wake up for a minimum of 10 minutes every hour.  For this lab, you don't need all the power! Go to Admin-> Warehouses -> SERVICENOW_WAREHOUSE -> ... > Edit, and change this to an XSMALL, and the auto timeout to one minute.
+
+> aside positive
+> Absolutely attach a resource monitor to the SERVICENOW_WAREHOUSE. Go to Admin->Resource Monitors->+ Resource Monitor, and create a warehouse resource monitor:
+![resource monitor](assets/monitor.png)
+
+## Select ServiceNow Tables
+Duration: 4
+
+> aside negative
+> A couple of things to be aware of:
+> - The connector can only ingest tables with **sys_id** columns present.
+> - ServiceNow views are not supported. Instead of ingesting these views, you should synchronize all tables for the underlying view and join the synchronized tables in Snowflake.
+> - Incremental updates occur only for tables with **sys_updated_on** or **sys_created_on** columns.
+> - For tables that do not have sys_updated_on or sys_created_on columns, the connector uses **truncate and load** mode.
+
+1. In the **Snowflake Connector for ServiceNow** window, select **Select Tables**.
+
+1. To be able to run our test query later, we need to ingest a couple of tables. From the search window enter **incident** and check the box next to it and choose a 24 hour sync time. 
+
+1. To choose other tables, clear the search, put the table name and select the checkbox. Do this for the following tables:
+
+   * sys_audit_delete
+   * task
+
+ > aside positive
+ >   Hint: Clear the search fields, and then select the title **Status** to sort and show all the tables you selected.
+
+ ![Select](assets/Select.png)
+
+
+1. Select **Start Ingestion**. The select windows closes and you get the message "Loading Data" from the main Connector window. In addition to the tables you choose, three system tables will also be loaded. These are necessary to build the views on the raw data: sys_dictionary, sys_db_object, and sys_glide_object.
+
+![load](assets/load.png)
+
+You receive a message indicating success:
```

Comment:
Should we add explanation when this message is displayed? It appears once at least one table has been fully ingested